ivy/functional/backends/numpy/linear_algebra.py

```diff
@@ -333,6 +333,10 @@ def vector_norm(
 # ----- #
 
 
+def adjoint(x: np.ndarray, /, *, out: Optional[ivy.Array] = None) -> np.ndarray:
+    return np.conj(x.transpose(list(range(x.ndim - 2)) + [x.ndim - 1, x.ndim - 2]))
```

Comment:
this can be simplified to `return np.moveaxis(np.conj(x), -1, -2)` similarity for JAX too, otherwise lgtm!